src/pages/components/icon-button.mdx

```diff
@@ -1,15 +1,117 @@
 ---
 title: Icon Button
 description: >
-  Description will be here
+  Icon Buttons (like Buttons) let users take action. They are used for repeated or persistent
+  actions on a page and lack labels to simplify the UI.
+package: '@zendeskgarden/react-buttons'
+components:
+  - buttons/src/elements/IconButton.tsx
 ---
 
-Content here
+<Usage>
+  <Use>
+    <ul>
+      <li>To simplify the appearance of repeated or persistent actions on a page</li>
+      <li>To enable action in a toolbar</li>
+    </ul>
+  </Use>
+  <Misuse>
+    <ul>
+      <li>To enable primary and secondary actions on a page, use a Button instead</li>
```

Comment:
In other examples, we link to the suggested components. Should we do the same here?